.gitignore

```diff
@@ -0,0 +1,3 @@
+node_modules
+
+test.md
```

Comment:
- [ ] Please I highly recommend adding `test.md` file with text `This file is ignored by git` per project requirements. 😄

Reply:
I have already added test.md file with the text and added this file in .gitignore so it can't be added in github repository.